src/api/routes/gem2s.js

```diff
@@ -0,0 +1,54 @@
+const AWSXRay = require('aws-xray-sdk');
+const { createGem2SPipeline } = require('../general-services/pipeline-manage');
+const ExperimentService = require('../route-services/experiment');
+const getBackendStatus = require('../general-services/backend-status');
+const pipelineResponse = require('../route-services/pipeline-response');
+const parseSNSMessage = require('../../utils/parse-sns-message');
+const logger = require('../../utils/logging');
+
+module.exports = {
+  'gem2s#get': (req, res, next) => {
+    getBackendStatus(req.params.experimentId)
+      .then((data) => res.json(data))
+      .catch(next);
+  },
+
+  'gem2s#create': (req, res, next) => {
```

Comment:
It is missing the authentication middleware that was added to the rest of endpoints while this was under development.

Reply:
added